app/main/forms.py

```diff
@@ -2563,3 +2580,23 @@ class PlatformAdminSearch(StripWhitespaceForm):
         param_extensions={"hint": {"text": ("Search for users, services, and organisations by name or partial name.")}},
         validators=[DataRequired()],
     )
+
+
+class PostcodeForm(StripWhitespaceForm):
+    postcode = PostcodeSearchField("Postcode")
+    radius = GovukDecimalField(
+        "Add radius",
+        param_extensions={
+            "classes": "govuk-input govuk-input--width-5",
+            "suffix": {"text": "km"},
+            "attributes": {"pattern": "[0-9]*"},
+        },
+        validators=[
+            NumberRange(min=0.099, max=38.001, message="Enter a radius between 0.1km and 38.0km."),
```

Comment:
Is there a particular reason we're limiting to 38km?

Reply:
The minimum value of 100m / 0.1km was driven by MNO requirements for the most effective transmission to get full penetration without risk of over subscription of cell towers. Too small an area might be difficult for their (MNO) systems to calculate which cell sites and towers to use, and 38km was driven by COBR as bigger than a city and the max side of alerting for non country size areas.